samples/DataSourceWithProcessingAndExtensions/Sample/SourceDataCookers/ProcessDataCooker.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using DataExtensionsSample.DataTypes;
+using DataExtensionsSample.Helpers;
+using DataExtensionsSample.OutputTypes;
+using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Extensibility.DataCooking;
+
+namespace DataExtensionsSample.SourceDataCookers
+{
+    public sealed class ProcessDataCooker
+        : SampleSourceDataCooker
+    {
+        public const string Identifier = "Process";
+        public static readonly DataCookerPath CookerPath = new DataCookerPath(CustomSourceParser.SourceId, Identifier);
```

Comment:
>public static readonly DataCookerPath CookerPath = new DataCookerPath(CustomSourceParser.SourceId, Identifier); [](start = 8, length = 111)

Why does EventStatsCooker have the CookerPathAsString that gets used by the EventStatsTable, but this cooker only has this CookerPath that gets used? Can we explain here and in the EventsStatsCooker why we define each of these fields?